apps/rush-lib/src/api/RushConfigurationProject.ts

```diff
@@ -216,6 +219,14 @@ export class RushConfigurationProject {
     return this._rushConfiguration;
   }
 
+  /**
+   * This index of this project in the Rush configuration's projects array
+   * @internal
+   */
+  public get _index(): number {
```

Comment:
I know we already have the getter convention, but it's probably fine to just create this as a `public readonly` property.

We can realistically convert the rest to that, but I'd want to do that in either a separate chagne or during the next major release.